Chandra/cmd_states/cmd_states.py

```diff
@@ -409,9 +409,9 @@ def get_state0(date=None, db=None, date_margin=10, datepar='datestop'):
     :rtype: dict
     """
     if db is None:
-        db = Ska.DBI.DBI(dbi='sybase', server='sybase', user='aca_read',
-                         database='aca')
-
+        db = Ska.DBI.DBI(dbi='sqlite',
```

Comment:
Not sure if this is tested

Reply:
I tested this manually and just confirmed that get_state0 seems to work fine with no db set in the code in this PR.